### ISSUE TITLE
Slayer counter not working/Jad plugin not working

### ISSUE THREAD

**Issue by geckofist** [runelite/runelite#6809]
**Describe the bug**
Essentially my slayer counter does not show up no matter how many times I refresh it or reset it. Additionally while doing Jad that plugin was not working either. It seems most other things are. 

**To Reproduce**
1. Open runelite, do not see slayer task
2. Attack monster that you've been assigned, still no slayer task

Additionally, it will not highlight my slayer targets. I've gone through three tasks now.

3. Get to Jad, no prayer helpers

**Screenshots**

**Environment (please complete the following information):**
 - OS: macOS
 - RuneLite version: Most up to date
 - Launcher version: Most up to date

**Comment by 15987632** [comment]
The prayer helped only works for jad when jad is rendered on your screen. Check your task on your slayer helm, ring, or gem to make sure it really is broken.

**Comment by geckofist** [comment]
Yeah, I had seen a few past issues and I made sure that Jad was rendered - I was directly fighting him.

I've confirmed each time that it is the proper task, as well as doing it (for example now, hellhounds), in the slayer cave where you must have the task to fight those animals.
